oe_theme.theme

```diff
@@ -1280,6 +1280,14 @@ function oe_theme_preprocess_pattern_link__preview(array &$variables): void {
   $variables['url'] = Url::fromUserInput((string) $variables['url']);
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_pattern_page_header__preview(array &$variables): void {
+  // We are going to accept only local images for the background image of the preview.
+  $variables['background_image_url'] = base_path() . drupal_get_path('module', 'oe_theme_helper')  . '/images/' . (string) $variables['background_image_url'];
```

Comment:
```suggestion
  $variables['background_image_url'] = base_path() . drupal_get_path('module', 'oe_theme_helper')  . '/images/' . $variables['background_image_url'];
```

We don't need casting as we are preprocessing the pattern preview, so we know it's a string.